### DISCUSSION TITLE
appcast stanza question

### DISCUSSION THREAD

**Discussion by GETandSELECT** [Homebrew/discussions#173] [Casks] [answered]
Hi

What exactly is the `appcast` stanza for? I thought this would automatically record new releases in cask, but that doesn't seem to be the case.

example: https://github.com/Homebrew/homebrew-cask/blob/master/Casks/ripme.rb#L6

There is a new release of this cask. I need to do a PR? In `appcast` linked feed the new release appears.

thank you

**Comment by gromgit** [comment, marked as answer]
In a nutshell, the content retrieved from the `appcast` URL is parsed by `brew` to determine the latest release version of the software. That's pretty much it.

As for the updating of the cask itself, it seems to still be a manual process.

See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md for the full details, and https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/all_stanzas.md for links to all the stanzas available.
